src/manifest.rs

```diff
@@ -99,61 +99,102 @@ pub(crate) struct LintTable {
     #[serde(default, deserialize_with = "deserialize_workspace_key")]
     pub(crate) workspace: bool,
     /// individual `lint_name = ...` entries
-    #[serde(flatten, deserialize_with = "deserialize_into_overridemap")]
-    pub(crate) inner: OverrideMap,
+    #[serde(flatten)]
+    pub(crate) inner: BTreeMap<Identifier, OverrideConfig>,
 }
 
-impl From<LintTable> for OverrideMap {
-    fn from(value: LintTable) -> OverrideMap {
-        value.inner
+impl LintTable {
+    /// Converts this into a stack of `OverrideMap`s, where the top of the stack (later
+    /// indices in the `Vec`) has higher priority than the bottom.
+    pub(crate) fn into_stack(self) -> Vec<OverrideMap> {
+        // use a priority -> OverrideMap BTreeMap, which will be sorted by priority
```

Comment:
Are the two uses of "priority" between the doc comment and the subsequent regular comment inside the function using opposite definitions of "higher priority" here, in terms of whether that signifies precedence vs lack thereof?

If so, that's going to be extremely confusing!